.github/workflows/update-homebrew-formula-on-release.yml

```diff
@@ -0,0 +1,19 @@
+name: Update Homebrew Formula On Release
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
We probably don't want to push pre-release tags 🤔

Reply:
No sure if we should use `types: [released]` or `types: [published]` here!

Reply:
Seems we went for `released`?

Reply:
@vincepri [docs](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release) is not very clear about this, but I think that "published" is trigger even if you publish a "draft" release.